indico/modules/events/abstracts/templates/abstract/timeline.html

```diff
@@ -144,10 +152,12 @@
                                 {%- endtrans %}
                             {% endif %}
                         {% endif %}
-                    </div>
-                    <div class="abstract-comment-score">
                         {% if review.score is not none %}
-                            {{ render_score(review.score, review.abstract.event_new.cfa) }}
+                            · score {{ render_score(review.score, review.abstract.event_new.cfa, classes='bold semantic-text') }}
+                            (<a class="js-ratings-toggle">{#--#}
```

Comment:
It's less ugly that way. Since whitespace is stripped on both the left `{#-` and right `-#}` side of the jinja comment there is no need to actually comment out the linebreak it self.